src/Onboarding/OnboardingScreen.tsx

```diff
@@ -168,20 +167,7 @@ const PositionDots: FunctionComponent<PositionDotsProps> = ({
   )
 }
 
-const skipButtonSpacingTop = Platform.select({
-  ios: Spacing.xSmall,
-  android: Spacing.xxLarge,
-})
-
-const languageButtonSpacingTop = Platform.select({
-  ios: Spacing.small,
-  android: Spacing.huge,
-})
-
-const headerFlex = Platform.select({
-  ios: 0.1,
-  android: 0.15,
-})
+const headerHeight = Platform.select({ ios: 65, android: 90, default: 65 })
```

Comment:
can we add names for these numbers?

Reply:
I'm kinda on the fence here. Is it not clear enough as is?

Reply:
Like, we could have:

```
const iosHeaderHeight = 65
const androidHeaderHeight = 65
const headerHeight = Platform.select({ ios: iosHeaderHeight, android: androidHeaderHeight, default: iosHeaderHeight )}
```

But I'm not sure I like that more.

Reply:
yeah a bit of overkill in this case, so totally your call and fine either way.

personally, I tend towards choosing to be black n white in gray areas as it tends to encourage better practices in the long term, so even though its kinda dumb to used a named constant for the number in this case, because using named constants for numbers is good idea in general, its makes sense to do it even in the 1 in 100 cases where its a bit too much.

Reply:
That makes sense! I'm down with the change.